ActionGame/ActionGame/Script/Manager/GameManager.h

```diff
@@ -7,10 +7,34 @@
 
 #include "../Character/Player.h"
 
+struct SaveState{
+	float x;
+	float y;
+	int stageNumber;
+};
+
 class GameManager : public Singleton<GameManager>{
 public:
 	GameManager(){};
 	~GameManager(){};
+
+public:
+	// ステージをテキストファイルから読み込む
+	void LoadStage();
+
+	std::vector<std::vector<ObjectTag>> GetStageData( int stageNumber_ );
+
+	int GetStageMax() const { return stageData.size(); }
+
+	// セーブ
+	void DataSaving( SaveState data_ );
+
+	// ロード
+	SaveState DataLoading();
```

Comment:
メソッド名は動詞始まりにして、
```suggestion
	// セーブ
	void SaveData( SaveState data_ );

	// ロード
	SaveState LoadData();
```
の方が良いですね。

ただ、 `SaveData()` だと所謂「セーブデータ」みたいに思えてイヤかも知れません。
いっそシンプルに  `Save()` `Load()` でも良いです。